pkg/adaptor/proxy/service.go

```diff
@@ -94,16 +102,20 @@ func (s *proxyService) getImageName(annotations map[string]string) (string, erro
 func (s *proxyService) CreateContainer(ctx context.Context, req *pb.CreateContainerRequest) (*types.Empty, error) {
 
 	logger.Printf("CreateContainer: containerID:%s", req.ContainerId)
-	if len(req.OCI.Annotations) > 0 {
-		logger.Print("    annotations:")
-		for k, v := range req.OCI.Annotations {
-			logger.Printf("        %s: %s", k, v)
-		}
-	}
```

Comment:
This change looks unrelated to the PR.

Reply:
Just adjust the print order. Because we may add new annotation when going through OCI Mounts, it's better print annotations after that

Reply:
Ok